README.md

```diff
@@ -14,30 +14,183 @@ jobs:
 
     strategy:
       matrix:
-        tutor_version: ["<17.0.0", "==17.0.3", "<18.0.0"]
+        tutor_version: ["<17.0.0", "==17.0.3", "<18.0.0", "nightly"]
 
     steps:
-      - uses: actions/checkout@v4
-        with:
-          path: eox-hooks
-
-      - uses: eduNEXT/integration-test-in-tutor
+      - name: Run Integration Tests
+        uses: eduNEXT/integration-test-in-tutor@main
         with:
+          app_name: "eox-test"
           tutor_version: ${{ matrix.tutor_version }}
-          app_name: "eox-hooks"
-          shell_file_to_run: "eox_hooks/tests/tutor/integration.sh"
+          shell_file_to_run: "tests/integration.sh"
+          openedx_extra_pip_requirements: "package1==1.0 package2>=2.0"
+          fixtures_file: "fixtures/test_data.json"
+          openedx_imports_test_file_path: "tests/integration/import_tests.py"
+          openedx_imports_test_function_name: "test_openedx_imports"
 ```
 
 ## Inputs
 
 ### `app_name`
 
-**Required** Application name to test. E.g., eox-tenant.
+**Required**  
+The name of your plugin/application to test. This should match the directory name of your plugin.  
+*Example*: `"eox-tenant"`
 
 ### `tutor_version`
 
-**Required** The tutor version matrix to use.
+**Required**  
+The version of Tutor to use. You can specify:
+
+- A specific version number (e.g., `"==17.0.3"`).
+- A comparison operator with a version (e.g., `"<18.0.0"`).
+- The string `"nightly"` to use the latest development version.
+
+*Examples*:
+
+- `"==17.0.3"`
+- `"<18.0.0"`
+- `"nightly"`
 
 ### `shell_file_to_run`
 
-(Optional) The path of the shell file to run the integration tests.
+**Required**  
+The path to the shell script that runs your integration tests. This path is relative to your plugin directory.  
+*Example*: `"tests/integration.sh"`
+
+### `openedx_extra_pip_requirements`
+
+**Optional**  
+Extra pip requirements to install in Open edX. These are additional Python packages your plugin depends on. Provide them as a space-separated string.  
+*Default*: `""` (empty string)  
+*Example*: `"package1==1.0 package2>=2.0"`
+
+### `fixtures_file`
+
+**Optional**  
+The path to the fixtures file of your plugin to load initial data for the tests. This path is relative to your plugin directory.  
+*Example*: `"fixtures/test_data.json"`
+
+### `openedx_imports_test_file_path`
+
+**Optional**  
+The path to the Python file in your plugin that contains the test function for validating Open edX imports. This path is relative to your plugin directory.  
+*Example*: `"tests/import_tests.py"`
+
+### `openedx_imports_test_function_name`
+
+**Optional**  
+The name of the function in the specified file that executes the import tests for Open edX.  
+*Example*: `"test_openedx_imports"`
+
+## Description
```

Comment:
Thanks for the suggestion! I think starting with the Usage section is better in general because it lets users quickly understand how to implement the action without diving into detailed explanations right away. They can always refer to the Description section later for more information. What I like about this structure is that we basically prioritize actionable steps while still providing detailed information for those who want it.

Let me know what you think

Reply:
That makes sense. We could change the title from description to overview or something similar so the actual description of the ga is in the first couple of lines of the document.

Reply:
Agreed, I've changed the name to _Overview_. Thanks!